packages/regl-worldview/src/Worldview.js

```diff
@@ -153,7 +153,11 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
     if (!this._tick) {
       this._tick = requestAnimationFrame(() => {
         this._tick = undefined;
-        worldviewContext.paint();
+        try {
+          worldviewContext.paint();
+        } catch (e) {
+          console.error(e);
```

Comment:
maybe throw this if it doesn't match `(regl) context lost`? We don't necessarily want errors in this look to be thrown away if they would be useful. Otherwise users might see a black 3d panel and we'd have no error notices